presto-orc/src/main/java/com/facebook/presto/orc/zstd/ZstdJniDecompressor.java

```diff
@@ -0,0 +1,53 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.facebook.presto.orc.zstd;
+
+import com.github.luben.zstd.Zstd;
+import io.airlift.compress.Decompressor;
+import io.airlift.compress.MalformedInputException;
+
+import java.nio.ByteBuffer;
+
+import static java.lang.StrictMath.toIntExact;
+
+public class ZstdJniDecompressor
```

Comment:
why was there no decompressor class before?

Reply:
@rschlussel Previously it was inside `OrcZstdDecompressor`, I refactored that class to use `ZstdJniDecompressor`.